eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -15,15 +15,22 @@ parameters:
 steps:
   - ${{ if not(parameters.SkipCheckoutNone) }}:
       - checkout: none
-
+  - pwsh: |
```

Comment:
Why not use a template parameter that is default to empty but can be passed a value such as `$(azuresdk-github-pat)` in cases where we want to use it.

Reply:
Synced offline. We don't want to change the sparse-checkout for now for private repo.